src/Http/Requests/StoreMeetupRequest.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Blumilk\Meetup\Core\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class StoreMeetupRequest extends FormRequest
+{
+    public function authorize()
+    {
+        return true;
+    }
+
+    public function rules()
```

Comment:
```suggestion
    public function rules(): array
```

Please change it in other form requests in the same way.